### PR TITLE
Added feature: Ability to create a post and view most recent

### PR DESCRIPTION
**Pull request by aatifjiwani**
Created the Article Model
Created the new/create methods to handle new articles from the user
Created the new.html.erb page which contains the form to create a new article.

### CODE REVIEW

File: app/views/articles/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<%= render 'templates/navbar' %>
```
Comment (dbernheisel) [inline app/views/articles/new.html.erb]: Rendering your navbar is repeated in several templates.
What do you think of pulling this into your `views/layouts/application.html.erb`?

File: test/controllers/articles_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ArticlesControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```
Comment (dbernheisel) [inline test/controllers/articles_controller_test.rb]: It's best to get some sort of test to assert your pages are rendering correctly.

File: app/controllers/articles_controller.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        
+        if @article.save
+            redirect_to :controller => 'home', :action => 'index'
+        else
+            render 'new'
+        end
+    end
+    
+    private
+    def article_param
```
Comment (dbernheisel) [inline app/controllers/articles_controller.rb]: What do you think of using `article_params` or `permitted_params`? There's a couple of parameters you're accepting so I think a pluralized method name would fit better.

**Comment by dbernheisel** [comment]
Great start

File: app/views/articles/new.html.erb (hunk 1 above)
Comment (aatifjiwani) [reply app/views/articles/new.html.erb]: Ahhh, that's a much smarter way of doing it

File: test/controllers/articles_controller_test.rb (hunk 2 above)
Comment (aatifjiwani) [reply test/controllers/articles_controller_test.rb]: I'm going to get started on testing today!

File: app/views/templates/_navbar.html.erb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<header class="site-header" role="banner">
+  <div class="wrapper">
+      <p class="site-title" >Haxxor News</p>
+      <nav class="site-nav">
+        <div class="trigger">
+            <%= link_to 'Home', home_index_path, {:class => 
+    'page-link'} %>
```
Comment (efatsi) [inline app/views/templates/_navbar.html.erb]: nitpicky, but you probably don't need a newline here

File: app/controllers/articles_controller.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    def new
+        @pg_title = "Create Post | Haxxor News"
+        @article = Article.new
+    end
+    
+    def create
+        @article = Article.new(article_params)
+        
+        if @article.save
+            redirect_to :controller => 'home', :action => 'index'
```
Comment (efatsi) [inline app/controllers/articles_controller.rb]: could just be `redirect_to root_url`

**Pull request closed without merging**